### PR TITLE
Dependency updates for master

### PR DESCRIPTION
**Pull request by Lex-2008**
Update pcre from 8.44 to 8.45
Update openssl from 1.1.1k to 1.1.1l
Update diffutils from 3.7 to 3.8
Update openldap from 2.5.5 to 2.6.0
Update libcurl-hub from 7.77.0 to 7.79.1
Update git from 2.32.0 to 2.33.1
Update postgresql from 13.3 to 13.4
Update libcurl from 7.77.0 to 7.79.1

### CODE REVIEW

**Comment by Lex-2008** [comment]
@cf-bottom build it without diffutils, shall we?

**Comment by cf-bottom** [comment]
Sure, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/7258/)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/7258/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-7258/

**Comment by cf-bottom** [comment]
Sure, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/7259/)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/7259/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-7259/

**Comment by Lex-2008** [comment]
@cf-bottom build it with exotics this time

**Comment by cf-bottom** [comment]
Sure, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/7263/)

(with exotics)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/7263/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-7263/

**Comment by Lex-2008** [comment]
@cf-bottom build exotics again, plz

**Comment by cf-bottom** [comment]
Sure, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/7279/)

(with exotics)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/7279/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-7279/

File: deps-packaging/pcre/distfiles [hunk 1]
```diff
@@ -1 +1 @@
-aecafd4af3bd0f3935721af77b889d9024b2e01d96b58471bd91a3063fb47728  pcre-8.44.tar.gz
+4e6ce03e0336e8b4a3d6c2b70b1c5e18590a5673a98186da90d4f33c23defc09  pcre-8.45.tar.gz
```
Comment (craigcomstock) [inline deps-packaging/pcre/distfiles:1]: Does this actually work with the ftp site? Did you seed the cache with the file somehow?

See https://github.com/cfengine/buildscripts/pull/871

And maybe add 

```
https://cfhcable.dl.sourceforge.net/project/pcre/pcre/8.45/
```

in `source` for pcre.

Comment (craigcomstock) [reply deps-packaging/pcre/distfiles:1]: Actually, I am looking into if we can use these sourceforge URLs or not. It seems we would need to add a way to rename the file in deps-packages/get-pkg-src. Playing around with it now.

Comment (Lex-2008) [reply deps-packaging/pcre/distfiles:1]: > Does this actually work with the ftp site? Did you seed the cache with the file somehow?

I must admit I didn't check ftp myself, but guess it was up at one point. Somehow the file got into the cache, yes:
Screenshot_2021-11-10_14-21-03

**Comment by Lex-2008** [comment]
@cf-bottom build exotics

**Comment by cf-bottom** [comment]
Sure, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/7289/)

(with exotics)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/7289/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-7289/

File: deps-packaging/openssl/cfbuild-openssl.spec [hunk 2]
```diff
@@ -1,4 +1,4 @@
-%define openssl_version 1.1.1k
+%define openssl_version 1.1.1l
```
Comment (craigcomstock) [inline deps-packaging/openssl/cfbuild-openssl.spec:1]: `1l` has got to be the silliest version ever. Ease of use? :p

**Review by craigcomstock — APPROVED** [review]

**Comment by craigcomstock** [comment]
AIX PR you can pull in if green is https://github.com/cfengine/buildscripts/pull/877

Build in progress to check is [Build Status](https://ci.cfengine.com/job/pr-pipeline/7292/)

**Comment by craigcomstock** [comment]
I think this PR still needs

https://github.com/cfengine/buildscripts/pull/873

and

https://github.com/cfengine/buildscripts/pull/878

**Review by nickanderson — APPROVED** [review]

**Comment by Lex-2008** [comment]
superseded by https://github.com/cfengine/buildscripts/pull/882

**Pull request closed without merging**